docs/details/backend.dox

```diff
@@ -16,6 +16,74 @@ will return in an exception.
 
 =======================================================================
 
+\defgroup unified_func_addbackendlibrary af_add_backend_library
+
+\brief Register an ArrayFire library located in an arbitrary path for use
```

Comment:
I just added a test on this, and they do share the state (which makes sense since `af_add_backend_library` eventually calls `loadLibrary`, which returns the same handle when given the same path, and all `af_set_backend_library` does is change which handle is assigned to `AFSymbolManager`'s `activeBackend`. Thanks, I'll add this to the documentation indeed.